### ISSUE TITLE
replacement transformer doesn't work in kustomization.yaml 

### ISSUE THREAD

**Issue by ndemeshchenko** [fluxcd/kustomize-controller#396]
Hello,
given this kustomization.yaml in a tenant dir pointed by flux kustomoization

```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization
resources:
  - ../../base/dev-team

namespace: integrations
configMapGenerator:
- name: metas
  literals:
  - limits.cpu=16
  - limits.memory=24Gi

replacements:
- source:
    kind: ConfigMap
    name: metas
    fieldPath: data.[limits.memory]
  targets:
  - select:
      name: common
      kind: ResourceQuota
    fieldPaths:
    - spec.hard.[limits.memory]
 ```

and the following resourcequota.yaml inside overlay directory
```apiVersion: v1
kind: ResourceQuota
metadata:
  name: common
  namespace: $NAMESPACE
spec:
  hard:
    limits.cpu: "1"
    limits.memory: "1Gi"
    resourcequotas: "1"
```

if I run kustomize in the directory it successfully override limits, however, kustomizew controller doesn't overrides it.

I wasn't able to find it in source but is there even a support for `replacements` in kustomize controller?

---
kustomize controller version tried so far
- ghcr.io/fluxcd/kustomize-controller:v0.13.0
- ghcr.io/fluxcd/kustomize-controller:v0.13.2

**Comment by JaneLiuL** [comment]
Hi @ndemeshchenko 
kustomize-controller can use `spec.postBuild.substitute` to replace variable. Please check it in https://fluxcd.io/docs/components/kustomize/kustomization/

**Comment by ndemeshchenko** [comment]
Hi @JaneLiuL. Yes, indeed, I'm aware of that. However, as per my understanding, `postBuild` is part of Kustomization resource from flux API `kustomize.toolkit.fluxcd.io/v1beta1`, not for `kustomize.config.k8s.io/v1beta1`.

**Comment by stefanprodan** [comment]
Looks like the [ReplacementTransformer](https://github.com/kubernetes-sigs/kustomize/blob/master/plugin/builtin/replacementtransformer/ReplacementTransformer.go) is not loaded by kustomize-controller even if we've enabled all bultin transformers.

**Comment by stefanprodan** [comment]
This has been fixed in kustomize-controller v0.13.3 release, @SomtochiAma tested it with your example and works.
